### ISSUE TITLE
GGMAX - Enhancement (Hud sound option)

### ISSUE THREAD

**Issue by Necrym59** [TheGameCreators/GameGuruRepo#4883]
Is it possible to get a screen toggle sound for the huds?  So when they call a hud it can play a particular sound for that hud.
Open for discussion if anyone want to add their weight to this particular option.
![hud_sound](https://github.com/TheGameCreators/GameGuruRepo/assets/95015657/2ce0ed7c-5ce4-42a0-99d8-e4de162e3a96)

**Comment by MonkeyFrogStudio** [comment]
That sounds great to me, honestly. There's very few things we shouldn't be able to attach a sound to, if we wanted to. ;)

**Comment by extremestrategy** [comment]
Yeah, that sounds like a really good idea Necrym, I don't usually play games with the sound on as I prefer to listen to music while I play, but still it would be added immersion for the gamers who have there sound on.

**Comment by JCLEONITA** [comment]
sure i asked for this to ..talked directly to necrym
